Userland/Games/GameOfLife/Pattern.cpp

```diff
@@ -24,13 +23,13 @@ void Pattern::set_action(GUI::Action* action)
 
 void Pattern::rotate_clockwise()
 {
-    Vector<DeprecatedString> rotated;
-    for (size_t i = 0; i < m_pattern.first().length(); i++) {
+    Vector<String> rotated;
+    for (size_t i = 0; i < m_pattern.first().bytes_as_string_view().length(); i++) {
         StringBuilder builder;
         for (int j = m_pattern.size() - 1; j >= 0; j--) {
-            builder.append(m_pattern.at(j).substring(i, 1));
+            builder.append(m_pattern.at(j).bytes_as_string_view().substring_view(i, 1));
         }
-        rotated.append(builder.to_deprecated_string());
+        rotated.append(builder.to_string().release_value_but_fixme_should_propagate_errors());
```

Comment:
Given that we know this is a small allocation and bearing #20405 in mind, should `.release_value_but_fixme_should_propagate_errors()` the best choice here or would `.release_value()` or `MUST` be better?

Reply:
I would go for MUST